Formula/hledger.rb

```diff
@@ -43,3 +50,12 @@ def install
     system "#{bin}/hledger-ui", "--version"
   end
 end
+
+# notes
+# "--no-install-ghc", "--system-ghc",  # without downloading ghc; use the one in $PATH, or fail if unsuitable
+# Try to adjust stack's --verbosity. Not much help; no progress output
+# for minutes if downloading ghc.
+# "--verbosity=debug",               # way too much, yet still not enough
+# "--verbosity=info",                # show detailed build progress (default)
+# "--verbosity=warn",                # show all deps' build warnings
+# "--verbosity=error",               # still shows warnings, can hide useful install plan failure details
```

Comment:
If you want you can make this a PR comment. Please don't comment in the formula unless it's to describe a patch

Reply:
Ok, are you requiring all comments to be removed ? I ask for clarity, because this is a non-routine human-maintained formula, and these notes can help the next maintainer (who might no be me). Also I didn't see anything about it in the homebrew contrib docs.